### ISSUE TITLE
Copyright

### ISSUE THREAD

**Issue by juliacmcf** [diamm/diamm#516]
We seem to have completely lost any form of copyright statement for manuscripts. Can this be resumed as a matter of some urgency? One of the libraries commented on it, but fortunately not rudely! The copyright statement tells the user who to approach for permission to use the images, but also reminds them that they are copyright...

The statement was originally attached to a library, but having found a couple of libraries where different MSS were owned by different people, the statement was attached to the MS instead. It should be in the original data. We need to put up a new MS with copyright statement on all images, so ideally the © should appear at the foot of the image viewer. (Ideally).

**Comment by ahankinson** [comment]
The copyright statement is still on the Archive record, but there are no longer any individual Source copyright statements. 

I can reinstate the Archive copyright statement on the pages, if you think that will be OK in the general case? Where would you like it to appear? On the image tab? Presumably you don't want to give a copyright statement that covers the source record, since the archive doesn't own the copyright on that.

**Comment by juliacmcf** [comment]
Yes, we need to give the copyright owner on the images. I think in a past iteration (pre Diva) the bottom part of the image-viewer frame had the copyright statement in view regardless of the image view or scrolling.

**Comment by ahankinson** [comment]
Done.
